ansible/roles/test/files/acstests/everflow_policer_test.py

```diff
@@ -168,6 +170,8 @@ def checkOriginalFlow(self):
 
         masked_exp_pkt = Mask(exp_pkt)
         masked_exp_pkt.set_do_not_care_scapy(scapy.Ether, "dst")
+        if self.check_ttl == 'False':
```

Comment:
Why is this a string instead of a bool?

Reply:
@daall i think parameter from pytest to ptftest will get pass as string. I am not sure though

Reply:
That makes sense, I missed that this was a ptftest file. :smile: